lighthouse/tests/beacon_node.rs

```diff
@@ -1313,8 +1312,8 @@ fn enr_tcp6_port_flag() {
 #[test]
 fn enr_match_flag_over_ipv4() {
     let addr = "127.0.0.2".parse::<Ipv4Addr>().unwrap();
-    let udp4_port = unused_udp4_port().expect("Unable to find unused port.");
-    let tcp4_port = unused_tcp4_port().expect("Unable to find unused port.");
+    let udp4_port = 0;
+    let tcp4_port = 0;
```

Comment:
this one is interesting, we should probably discourage users from doing this, or change the behaviour: a match enr with zero port should either not be accepted or mean: set in the enr whatever the os gives us

Reply:
so here I mean, this is testing wrong behaviour but we need to a) deal with it and fix later or b) use a specific port that is not zero. I'm fine with both